events/schema.py

```diff
@@ -7,6 +8,11 @@
 from venues.schema import VenueNode
 
 
+class EventTranslationNode(DjangoObjectType):
+    class Meta:
+        model = apps.get_model("events", "EventTranslation")
+
```

Comment:
Good point. I thought we still need `id` or `master` to find a translation when updating it, but since a translation of an event can also be identified using `languageCode` so it's fine to just exclude these fields